core/src/com/unciv/logic/civilization/CivilizationInfo.kt

```diff
@@ -365,9 +365,16 @@ class CivilizationInfo {
 
     fun isAtWar() = diplomacy.values.any { it.diplomaticStatus == DiplomaticStatus.War && !it.otherCiv().isDefeated() }
 
+    /**
+     * Returns a civilization caption suitable for greetings including player type info:
+     * Like "Milan" if the nation is a city state, "Caesar of Rome" otherwise, with an added
+     * " (AI)", " (Human - Hotseat)", or " (Human - Multiplayer)" if the game is multiplayer.
+     */ 
     fun getLeaderDisplayName(): String {
         return nation.getLeaderDisplayName().tr() +
             when {
+                !gameInfo.gameParameters.isOnlineMultiplayer ->
```

Comment:
???

Reply:
Quote: "getLeaderDisplayName adds AI/Human only in actual multiplayer games. As single player I don't need to know all the others are AI's? Was I right in thinking that?" - so I was already unsure - and rightfully so...

<details><summary>Warning: Explanation as pictorial!</summary>

(original)
![image](https://user-images.githubusercontent.com/63000004/121196235-9b294e80-c870-11eb-8e0d-4cdb09710bb0.png)
(this branch)
![image](https://user-images.githubusercontent.com/63000004/121196319-ab412e00-c870-11eb-8ecf-458908d60bd6.png)
(original)
![image](https://user-images.githubusercontent.com/63000004/121196342-b300d280-c870-11eb-8c93-4e86ce5c4f0f.png)
</details>

<details><summary>So I came up with:</summary>

```kotlin
    fun getLeaderDisplayName(): String {
        /* Logic:
            We don't need to know AI/Human if:
                - It is a city state, recognizable via its nation icon?
                - We're single player: !isOnlineMultiplayer && Human.count==1
                - We're displaying the spectator
            We want to know AI/Human if:
                - We're online multiplayer
                - We're hotseat multiplayer
                - We're spectator????? -> those are counted as human / this doesn't know who's looking
         */
        val severalHumans = gameInfo.civilizations.count { it.playerType == PlayerType.Human } > 1
        val online = gameInfo.gameParameters.isOnlineMultiplayer
        return nation.getLeaderDisplayName().tr() +
            when {
                nation.name == Constants.spectator ->
                    ""                      // everybody knows the spectator is always human
                !online && !severalHumans ->
                    ""                      // offline single player will know everybody else is AI
                isCityState() ->
                    ""                      // City states are known to always be AI and are recognizable
                playerType == PlayerType.AI ->
                    " (" + "AI".tr() + ")"
                online ->
                    " (" + "Human".tr() + " - " + "Multiplayer".tr() + ")"
                else ->
                    " (" + "Human".tr() + " - " + "Hotseat".tr() + ")"
            }
    }
```
</details>

... But I think I'll leave the spectator and citystate cases out. Who wants <sup><sub>(can)</sub></sup> meet a ghost anyway?